src/List.ls

```diff
@@ -15,6 +15,13 @@ filter = (f, xs) -->
 reject = (f, xs) -->
   [x for x in xs when not f x]
 
+remove = (el, xs) -->
+  i = elem-index el, xs
+  results = xs.slice!
+  if i?
+    results.splice i, 1
+  results
```

Comment:
I'm happy to use the fancy cascade. :smiley: I generally avoid them in my own code, as they seem to confuse people coming from both imperative *and* functional backgrounds, but they are undeniably cool!